pkg/transient-disk/transient-disk.go

```diff
@@ -0,0 +1,127 @@
+package transientdisk
+
+import (
+	"errors"
+	"fmt"
+
+	kubev1 "k8s.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	registrydisk "kubevirt.io/kubevirt/pkg/registry-disk"
+)
+
+const (
+	filePrefix              = "transient-disk-image"
+	fileType                = "qcow2"
+	transientDiskOwner      = "qemu"
+	mountBaseDir            = "/var/run/transientdisk"
+	transientContainerImage = "kubevirt/transient-disk:devel"
+)
+
+func generateVMBaseDir(vm *v1.VirtualMachine) string {
+	return fmt.Sprintf("%s", mountBaseDir)
+}
+func generateVolumeMountDir(vm *v1.VirtualMachine) string {
+	baseDir := generateVMBaseDir(vm)
+	return fmt.Sprintf("%s", baseDir)
+}
+
+func GetFilePath(vm *v1.VirtualMachine, volumeName string) (string, string, error) {
+	volumeMountDir := generateVolumeMountDir(vm)
+	return volumeMountDir + "/" + volumeName + ".qcow2", "qcow2", nil
```

Comment:
there's no reason to return an error here if 'nil' is the only possibility for error.

Reply:
This function actually tries to imitate GetFilePath from registrydisk (they could be behind interface).